packages/drivers/odsp-driver/src/odspSummaryUploadManager.ts

```diff
@@ -91,7 +92,7 @@ export class OdspSummaryUploadManager {
             entries: snapshotTree.entries!,
             message: "app",
             sequenceNumber: referenceSequenceNumber,
-            type: "channel",
+            type: initialSummary ? "container" : "channel",
```

Comment:
Some comment somewhere here (actually better would be where ISummaryContext.initialSummary defined - see my earlier comment) would be great to describe intentions, i.e. what this flag means, pre-conditions, expectations.